src/locale/fr-FR.h

```diff
@@ -0,0 +1,124 @@
+// Copyright 2019 - Mathieu D(@Knackie)
+// Locale/language file for French / Quebec.
+// This file will override the default values located in `defaults.h`.
+#ifndef LOCALE_FR_FR_H_
+#define LOCALE_FR_FR_H_
+
+#define D_STR_UNKNOWN "INCONNU"
+#define D_STR_PROTOCOL "Protocole"
+#define D_STR_ON "On"
+#define D_STR_OFF "Off"
+#define D_STR_MODE "Mode"
+#define D_STR_TOGGLE "Bascule"
+#define D_STR_SLEEP "Pause"
+#define D_STR_LIGHT "Lumière"
+#define D_STR_POWERFUL "Puissance"
+#define D_STR_QUIET "Silence"
+#define D_STR_ECONO "Economie"
+#define D_STR_BEEP "Bip"
+#define D_STR_MOLD "Moule"
+#define D_STR_CLEAN "Nettoyer"
+#define D_STR_PURIFY "Purifier"
+#define D_STR_TIMER "Timer"
+#define D_STR_ONTIMER D_STR_ON " " D_STR_TIMER
+#define D_STR_OFFTIMER D_STR_OFF " " D_STR_TIMER
+#define D_STR_CLOCK "Heure"
+#define D_STR_COMMAND "Commandement"
+#define D_STR_HEALTH "Santé"
+#define D_STR_TEMP "Temporaire"
+#define D_STR_HUMID "Humidité"
+#define D_STR_SAVE "Sauvegarder"
+#define D_STR_EYE "Oeil"
+#define D_STR_FOLLOW "Suivre"
+#define D_STR_FRESH "Frais"
+#define D_STR_HOLD "Maintenir"
+#define D_STR_BUTTON "Bouton"
+#define D_STR_NIGHT "Nuit"
+#define D_STR_SILENT "Silence"
+#define D_STR_UP "En haut"
+#define D_STR_TEMPUP D_STR_TEMP " " D_STR_UP
+#define D_STR_DOWN "En bas"
+#define D_STR_TEMPDOWN D_STR_TEMP " " D_STR_DOWN
+#define D_STR_CHANGE "Changement"
+#define D_STR_MOVE "Move"
+#define D_STR_SET "Mettre"
+#define D_STR_CANCEL "Annuler"
+#define D_STR_COMFORT "Cpnfort"
+#define D_STR_WEEKLY "Chaque semaine"
+#define D_STR_WEEKLYTIMER D_STR_WEEKLY " " D_STR_TIMER
+#define D_STR_FAST "Rapide"
+#define D_STR_SLOW "Lent"
+#define D_STR_AIRFLOW "Ebauche"
+#define D_STR_STEP "Etape"
+#define D_STR_NA "N/A"
+#define D_STR_OUTSIDE "Plein air"
+#define D_STR_LOUD "Fort"
+#define D_STR_UPPER "Au dessus"
+#define D_STR_LOWER "En dessous"
+#define D_STR_BREEZE "Brise"
+#define D_STR_CIRCULATE "Faire circuler"
+#define D_STR_CEILING "Plafond"
+#define D_STR_WALL "Mur"
+#define D_STR_ROOM "Pièce"
+#define D_STR_6THSENSE "6ter Sens"
+#define D_STR_FIXED "Fixer"
+
+#define D_STR_AUTOMATIC "Automatique"
+#define D_STR_MANUAL "Manuel"
+#define D_STR_COOL "Frais"
+#define D_STR_HEAT "Chaleur"
+#define D_STR_FAN "Ventillateur"
+#define D_STR_FANONLY "Seul_fan"
+#define D_STR_DRY "Sec"
+
+#define D_STR_MED "Avec"
+#define D_STR_MEDIUM "Moyen"
+
+#define D_STR_HIGHEST "Le plus haut"
+#define D_STR_HIGH "Haut"
+#define D_STR_HI "H"
+#define D_STR_MID "M"
+#define D_STR_MIDDLE "Moitié"
+#define D_STR_LOW "Bas"
+#define D_STR_LO "B"
+#define D_STR_LOWEST "Le plus bas"
+#define D_STR_RIGHT "Droite"
+#define D_STR_MAXRIGHT D_STR_MAX " " D_STR_RIGHT
```

Comment:
We are probably going to need to define "D_STR_MAX"
e.g.
`#define D_STR_MAX "Max"`